src/clientSideScene/sceneEntities.ts

```diff
@@ -98,8 +98,9 @@ export const TANGENTIAL_ARC_TO__SEGMENT_DASH =
   'tangential-arc-to-segment-body-dashed'
 export const TANGENTIAL_ARC_TO_SEGMENT = 'tangential-arc-to-segment'
 export const TANGENTIAL_ARC_TO_SEGMENT_BODY = 'tangential-arc-to-segment-body'
-export const MIN_SEGMENT_LENGTH = 60 // in pixels
 export const SEGMENT_WIDTH_PX = 1.6
+export const HIDE_SEGMENT_LENGTH = 60 // in pixels
+export const HIDE_HOVER_SEGMENT_LENGTH = 40 // in pixels
```

Comment:
```suggestion
export const HIDE_SEGMENT_LENGTH = 75 // in pixels
export const HIDE_HOVER_SEGMENT_LENGTH = 60 // in pixels
```

Yeah after trying it I agree with your comment, that bumping these up helps with click targeting. I'm also cool with discarding this additional logic if it feels like too finicky for you; it doesn't to me.

Reply:
Nah not too finicky.